pqarrow/arrowutils/sort.go

```diff
@@ -0,0 +1,124 @@
+package arrowutils
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"sort"
+
+	"github.com/apache/arrow/go/v14/arrow"
+	"github.com/apache/arrow/go/v14/arrow/array"
+	"github.com/apache/arrow/go/v14/arrow/compute"
+	"github.com/apache/arrow/go/v14/arrow/memory"
+	"golang.org/x/exp/constraints"
```

Comment:
```suggestion
```

Reply:
No need to pull up this constraint. It contains `~string` while we only use `string`.